### PR TITLE
Add several tools  for release

### PR DESCRIPTION
**Pull request by weiguoz**
add several tools like(gpg,javadoc..) to release the pkg to sonatype.org central

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,8 +7,9 @@
     <groupId>org.sqlflow</groupId>
     <artifactId>jsqlflow</artifactId>
     <packaging>jar</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>0.0.1</version>
```
Comment (typhoonzero) [inline pom.xml:10]: maybe the version start from `0.1.0`?

Comment (weiguoz) [reply pom.xml:10]: Agree.
The `pb` is being iterated. Let's update the version to `0.1.0` when the protocol is stabled.

File: pom.xml [hunk 2]
```diff
@@ -7,8 +7,9 @@
     <groupId>org.sqlflow</groupId>
```
Comment (Yancey0623) [inline pom.xml:7]: Just confusion about the `groupId` field, should it be `sqlfow.org` ?

Comment (weiguoz) [reply pom.xml:7]: [Java conventions](https://maven.apache.org/guides/mini/guide-naming-conventions.html)

File: .travis.yml [hunk 3]
```diff
@@ -3,12 +3,18 @@ sudo: false
...
   - JDK=oraclejdk8
 dist: trusty
 before_script:
   - jdk_switcher use $JDK
 
-script:
-  - mvn clean verify --batch-mode --errors --fail-at-end --show-version -DinstallAtEnd=true
+install: mvn clean verify -Dgpg.skip
+
+# TODO(weiguo) not tested yet
```
Comment (weiguoz) [inline .travis.yml:14]: In fact, it will fail to run the script `mvn ... ` due to the incorrect `gpg` configuration.
I left an issue #13

**Review by typhoonzero — APPROVED** [review]
LGTM

**Pull request merged**